### ISSUE TITLE
The current app appears to not be working on several devices

### ISSUE THREAD

**Issue by unioncapitalboston** [drenfr01/unionCapital#372]
Maybe it is an Android/Chrome issue? I can't get it to load on my MacBook Chrome:
![messages image 1931846893](https://cloud.githubusercontent.com/assets/7445103/16028376/ba5032a2-31ac-11e6-8e25-e8eb7e93382a.png)

**Comment by drenfr01** [comment]
Since Drew was kind enough to fix it, I'll pass on the explanation (courtesy of Drew). 

Basically, we use a package (i.e. snippet of code that someone else wrote and released for anyone to use) to map routes in the browser, e.g. /login, to specific pieces of app functionality. Unfortunately, the package we use is no longer being maintained (i.e. the person who made it / supported it for several years hasn't touched it in ~6-8 months). Since the person no longer supports it, a new release of Chrome caused it to break (side note: nice call Eric, it was Chrome that caused the issue). That's what Drew fixed. 

The good news is that there is a replacement package which is maintained, and this time it looks to have official support (i.e. the makers of Meteor, the technology we used, have committed to supporting it.)

The bad news is that ripping out this package and putting in the new one is a large amount of effort. 

Long story short, this is a known piece of technical debt we should start working on after this next release.

**Comment by drenfr01** [comment]
@unioncapitalboston @ballek I believe we can close this?

**Comment by ballek** [comment]
@drenfr01 indeed... CLOSED
